easybuild/easyblocks/r/rust.py

```diff
@@ -81,38 +114,11 @@ def configure_step(self):
 
     def install_step(self):
         """Custom install step for Rust"""
-
         super(EB_Rust, self).install_step()
 
         if build_option('rpath'):
-            # make sure that all shared libraries use RPATH, not RUNPATH;
-            # cfr. https://github.com/easybuilders/easybuild-easyconfigs/issues/18079
-            shlib_ext = get_shared_lib_ext()
-            shared_libs = glob.glob(os.path.join(self.installdir, 'lib', f'lib*.{shlib_ext}'))
-            executables = glob.glob(os.path.join(self.installdir, 'bin', '*'))
-            binary_files = [file for file in executables + shared_libs if is_binary(read_file(file, mode='rb'))]
-
-            for bin_file in binary_files:
-                res = run_shell_cmd(f"readelf -d {bin_file}", hidden=True)
-                if res.exit_code:
-                    raise EasyBuildError(f"Failed to check RPATH section in {bin_file}: {res.output}")
-
-                if RUNPATH_PATTERN.search(res.output):
-                    self.log.debug(f"RUNPATH section found in {bin_file} - need to change to RPATH")
-                    # determine current RUNPATH value
-                    res = run_shell_cmd(f"patchelf --print-rpath {bin_file}")
-                    if res.exit_code:
-                        raise EasyBuildError(f"Failed to determine current RUNPATH value for {bin_file}: {res.output}")
-
-                    # use RUNPATH value to RPATH value
-                    runpath = res.output.strip()
-                    res = run_shell_cmd(f"patchelf --set-rpath '{runpath}' --force-rpath {bin_file}")
-                    if res.exit_code:
-                        raise EasyBuildError(f"Failed to set RPATH for {bin_file}: {res.output}")
-
-                    self.log.info(f"RUNPATH converted to RPATH in file: {bin_file}")
-                else:
-                    self.log.debug(f"No RUNPATH section found in {bin_file}")
+            if LooseVersion(self.version) >= LooseVersion('1.80.0'):
```

Comment:
This isn't good enough, the version check needs to be done in `_convert_runpaths_to_rpaths`, because now we're no longer fixing RUNPATH to RPATH for `lib/lib*.so` for say `Rust-1.79.0-GCCcore-13.3.0.eb`...